hax/setup.py

```diff
@@ -126,7 +126,8 @@ def get_motr_cflags():
     packages=find_packages(),
     setup_requires=['flake8', 'mypy', 'pkgconfig'],
     install_requires=[
-        'python-consul>=1.1.0', 'simplejson', 'aiohttp', 'click', 'dataclasses'
+        'python-consul>=1.1.0', 'simplejson', 'aiohttp', 'click', 
```

Comment:
Format the sources with [`yapf`](ssc-vm-2492.colo.seagate.com) please. I guess that would resolve Codacy issue here.